package.json

```diff
@@ -33,7 +33,8 @@
     "api-stubs:reset": "npx ts-node --transpile-only ./wiremock/resetStubs.ts",
     "start-test-wiremock": "docker-compose -f docker-compose-test.yml up -d",
     "generate-types": "script/generate-types",
-    "form-page:generate": "npx ts-node --transpile-only ./server/form-pages/utils/generator.ts"
+    "form-page:generate": "npx ts-node --transpile-only ./server/form-pages/utils/generator.ts",
+    "schema:generate": "npx ts-node --transpile-only ./server/form-pages/utils/schemaGenerator.ts"
```

Comment:
Good shout - TBH, the form page generator could use some tweaking to support the new format. I dunno how useful it is anyway given we're not cranking out pages at the rate we once were - what do you think?

Reply:
Ah yeah good point. I don't think it's worth us spending any time on it at the moment. It would be useful in future but I can't see us using it much in the short term

Reply:
It's unexpected to have one that is `x:generate` and another that is `generate:x`